src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -13,8 +13,24 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <!-- Documentation warnings -->
     <NoWarn>$(NoWarn);1591</NoWarn>
+    <TargetsForTfmSpecificBuildOutput>GatherProjectReferenceAssets</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
-  
+
+  <!-- The following properties are set to package M.D.A.Interactive with the M.D.A nuget package. If M.D.A.I undergoes TFM or dependency changes, we need to update the dependencies below-->
+  <Target Name="GatherProjectReferenceAssets" Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+    <MSBuild Projects="./../Microsoft.Data.Analysis.Interactive/Microsoft.Data.Analysis.Interactive.csproj"
+             Targets="_GetBuildOutputFilesWithTfm"
+             Properties="TargetFramework=$(TargetFramework)">
+      <Output TaskParameter="TargetOutputs" ItemName="BuildOutputInPackage" />
+    </MSBuild>
+  </Target>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+    <PackageReference Include="HtmlAgilityPack" Version="1.11.24" />
+    <PackageReference Include="Microsoft.DotNet.Interactive" Version="1.0.0-beta.20154.7" />
```

Comment:
You'll want to update to a newer version. There are some breaking changes, outlined here: https://github.com/dotnet/interactive/issues/512.

Reply:
> You'll want to update to a newer version.

@jonsequitur, @eerhardt could you direct me to a newer version?
The latest nuget version is 1.0.0-beta.20154.7 (see  https://www.nuget.org/packages/Microsoft.DotNet.Interactive)
Thank you.